modular_skyrat/modules/emote_panel/code/emote_panel.dm

```diff
@@ -0,0 +1,978 @@
+		/mob/living/proc/emote_cross,
+		/mob/living/proc/emote_chuckle,
+		/mob/living/proc/emote_collapse,
+		/mob/living/proc/emote_cough,
+		/mob/living/proc/emote_dance,
+		/mob/living/proc/emote_drool,
+		/mob/living/proc/emote_faint,
+		/mob/living/proc/emote_flap,
+		/mob/living/proc/emote_aflap,
+		/mob/living/proc/emote_frown,
+		/mob/living/proc/emote_gag,
+		/mob/living/proc/emote_giggle,
+		/mob/living/proc/emote_glare,
+		/mob/living/proc/emote_grin,
+		/mob/living/proc/emote_groan,
+		/mob/living/proc/emote_grimace,
+		/mob/living/proc/emote_jump,
+		/mob/living/proc/emote_kiss,
+		/mob/living/proc/emote_laugh,
+		/mob/living/proc/emote_look,
+		/mob/living/proc/emote_nod,
+		/mob/living/proc/emote_point,
+		/mob/living/proc/emote_pout,
+		/mob/living/proc/emote_scream,
+		/mob/living/proc/emote_scowl,
+		/mob/living/proc/emote_shake,
+		/mob/living/proc/emote_shiver,
+		/mob/living/proc/emote_sigh,
+		/mob/living/proc/emote_sit,
+		/mob/living/proc/emote_smile,
+		/mob/living/proc/emote_sneeze,
+		/mob/living/proc/emote_smug,
+		/mob/living/proc/emote_sniff,
+		/mob/living/proc/emote_stare,
+		/mob/living/proc/emote_strech,
+		/mob/living/proc/emote_sulk,
+		/mob/living/proc/emote_sway,
+		/mob/living/proc/emote_tilt,
+		/mob/living/proc/emote_tremble,
+		/mob/living/proc/emote_twitch,
+		/mob/living/proc/emote_twitch_s,
+		/mob/living/proc/emote_wave,
+		/mob/living/proc/emote_whimper,
+		/mob/living/proc/emote_wsmile,
+		/mob/living/proc/emote_yawn,
+		/mob/living/proc/emote_gurgle,
+		/mob/living/proc/emote_inhale,
+		/mob/living/proc/emote_exhale,
+		/mob/living/proc/emote_swear
+	)
+	all_emotes += living_emotes
+	if(istype(src, /mob/living))
+		available_emotes += living_emotes
+
+	// code\modules\mob\living\carbon\emote.dm
+	var/list/carbon_emotes = list(
+		/mob/living/carbon/proc/emote_airguitar,
+		/mob/living/carbon/proc/emote_blink,
+		/mob/living/carbon/proc/emote_blink_r,
+		/mob/living/carbon/proc/emote_crack,
+		/mob/living/carbon/proc/emote_circle,
+		/mob/living/carbon/proc/emote_moan,
+		///mob/living/carbon/proc/emote_noogie,
+		/mob/living/carbon/proc/emote_slap,
+		/mob/living/carbon/proc/emote_wink
+	)
+	all_emotes += carbon_emotes
+	if(istype(src, /mob/living/carbon))
+		available_emotes += carbon_emotes
+
+	// code\modules\mob\living\carbon\human\emote.dm
+	var/list/human_emotes = list(
+		/mob/living/carbon/human/proc/emote_cry,
+		///mob/living/carbon/human/proc/emote_dap,
+		/mob/living/carbon/human/proc/emote_eyebrow,
+		/mob/living/carbon/human/proc/emote_grumble,
+		///mob/living/carbon/human/proc/emote_hug,
+		/mob/living/carbon/human/proc/emote_mumble,
+		/mob/living/carbon/human/proc/emote_pale,
+		/mob/living/carbon/human/proc/emote_raise,
+		/mob/living/carbon/human/proc/emote_salute,
+		/mob/living/carbon/human/proc/emote_shrug,
+		/mob/living/carbon/human/proc/emote_wag,
+		/mob/living/carbon/human/proc/emote_wing
+	)
+	all_emotes += human_emotes
+	if(istype(src, /mob/living/carbon/human))
```

Comment:
ishuman

Reply:
Appropriate check applied, also moved the checks to make it easier to read